<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>40</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Input file:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="inpBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Accepted formats: .hyp</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inpButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>50</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="picked_Radio">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>140</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Picked?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
